adafruit_epd/ssd1680.py

```diff
@@ -24,7 +24,7 @@
 except ImportError:
     pass
 
-__version__ = "0.0.0+auto.0"
+__version__ = "2.13.0"
```

Comment:
```suggestion
__version__ = "0.0.0+auto.0"
```

Reply:
Woops. I saw this line change when making the commit, but wasn't sure if it would be an issue. I'll look out for those in the future.